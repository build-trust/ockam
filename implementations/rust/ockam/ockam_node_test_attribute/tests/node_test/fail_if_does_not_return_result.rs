#[ockam_node_test_attribute::node_test]
async fn my_test(ctx: ockam::Context) -> String {
    ctx.address();
}
