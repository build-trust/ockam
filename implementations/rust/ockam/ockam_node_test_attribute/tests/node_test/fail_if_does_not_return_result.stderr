error: The test function must return a Result
 --> $DIR/fail_if_does_not_return_result.rs:2:7
  |
2 | async fn my_test(ctx: ockam::Context) -> String {
  |       ^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> $DIR/fail_if_does_not_return_result.rs:1:1
  |
1 | / #[ockam_node_test_attribute::node_test]
2 | | async fn my_test(ctx: ockam::Context) -> String {
3 | |     ctx.address();
4 | | }
  | |_^ consider adding a `main` function to `$DIR/tests/node_test/fail_if_does_not_return_result.rs`
